rocketchat-core/src/main/java/com/rocketchat/core/model/attachment/Attachment.java

```diff
@@ -8,26 +10,32 @@
 import org.json.JSONObject;
 
 /**
- * Created by sachin on 20/8/17.
+ * @author Sachin Shinde
+ * @author Filipe de Lima Brito (filipedelimabrito@gmail.com)
  */
-
 public class Attachment {
```

Comment:
Could you try to convert this to `AutoValue`?  You can look on the current app.

Don't forget to add a method like this
```java
    public static JsonAdapter<Attachment> jsonAdapter(Moshi moshi) {
        return new AutoValue_Attachment.MoshiJsonAdapter(moshi);
    }
```
so we can generate the Json Parser for it.